### PR TITLE
Initial-adding-repo-ui

### PR DESCRIPTION
**Pull request by ajayVibinex**

### CODE REVIEW

File: scripts/contentScript.js [hunk 1]
```diff
@@ -5,6 +5,48 @@ const keyToLabel = Object.freeze({
...
+async function cssForGithubTrackRepo(trackedRepos) {
+	const allOrgRepo = document.getElementById('org-repositories');
+	const ignoreUrl = [
+		'stargazers',
+		'forks',
+		'issues',
+		'pulls',
+		'issues?q=label%3A%22help+wanted%22+is%3Aissue+is%3Aopen',
+		'repositories?type=all',
+		'issues?q=label%3A%22Help+wanted%22+is%3Aissue+is%3Aopen'
```
Comment (avikalpg) [inline scripts/contentScript.js:17]: These three look very sensitive - it seems that they might not cover all the cases

File: scripts/contentScript.js [hunk 2]
```diff
@@ -5,6 +5,48 @@ const keyToLabel = Object.freeze({
...
+	];
+	const orgRepoUrl = Array.from(allOrgRepo.getElementsByTagName('a'));
+
+	orgRepoUrl.forEach((item) => {
+		const link = item.getAttribute('href').split('/');
+		const orgRepoName = link[link.length - 1];
+
+		if (ignoreUrl.includes(orgRepoName)) {
+		} else {
```
Comment (avikalpg) [inline scripts/contentScript.js:26]:
```suggestion
		if (!ignoreUrl.includes(orgRepoName)) {
```

Why do you want to keep an empty block?

File: scripts/contentScript.js [hunk 3]
```diff
@@ -62,42 +103,42 @@ function addCssElementToBitbucket(highlightedPRIds) {
...
-// fetching data from API 
-async function getDataFromAPI(repoOwner, repoName) {
-	const data = {
-		"repo_owner": `${repoOwner}`,
-		"repo_name": `${repoName}`,
-		"alias_list": ["tapish303@gmail.com", "tapish@vibinex.com", "tapish@iitj.ac.in"],
-		"is_github": true
-	}
-	let highlightedPRIds;
+async function apiCall(url, body) {
```
Comment (avikalpg) [inline scripts/contentScript.js:112]: Good! 👏🏼

File: scripts/contentScript.js (hunk 1 above)
Comment (ajayCreations) [reply scripts/contentScript.js:17]: I alreayd checked all these are same in every organization repo.

**Pull request closed without merging**